views/fightScreen.py

```diff
@@ -0,0 +1,622 @@
+from random import choice as randChoice
+from PySide6.QtCore import Qt, QTimer
+from PySide6.QtGui import QPixmap, QMovie, QColor
+from PySide6.QtWidgets import (
+    QLabel,
+    QWidget,
+    QGridLayout,
+    QSpacerItem,
+    QSizePolicy,
+    QPushButton,
+)
+from views.meter import Meter
+
+class FightScreen(QWidget):
+    def __init__(self):
+        super().__init__()
+
+        self.imageAssets = {
+                item: QPixmap(f"./assets/{item}.jpg")
+                for item in ['sword', 'spear', 'shield', 'gloves']
+                }
```

Comment:
@instMeyFTW, can you add the assets to the repo so we can use them?

Reply:
On it.

Reply:
Once these are added I think it's ready to merge.